assets/methods_description_template.yml

```diff
@@ -0,0 +1,25 @@
+id: "nf-core-circrna-methods-description"
+description: "Suggested text and references to use when describing pipeline usage within the methods section of a publication."
+section_name: "nf-core/circrna Methods Description"
+section_href: "https://github.com/nf-core/circrna"
+plot_type: "html"
+## TODO nf-core: Update the HTML below to your prefered methods description, e.g. add publication citation for this pipeline
```

Comment:
you might want to add your paper here

Reply:
I'll give it a go! I went to the Sarek one to copy but... ;)